livingatlas/pipelines/src/main/java/au/org/ala/clustering/HashKeyOccurrenceBuilder.java

```diff
@@ -0,0 +1,162 @@
+package au.org.ala.clustering;
+
+public final class HashKeyOccurrenceBuilder {
```

Comment:
Just curious - why not Lombok `@Builder` on the HashKeyOccurrence?

Reply:
Problems/conflicts with the Apache Beam JavaBean schema implementation. Sorry - I didn't make a note of the errors at the time, but I couldn't get it to work with lombok.